### ISSUE TITLE
[🚀 Feature Request ]: Support EMULATOR_SYS_IMG argument

### ISSUE THREAD

**Issue by Tpuljak** [budtmo/docker-android#362]
### Idea

It would be great if the SYS_IMG could be passed as an argument.

This would allow, for example, `armeabi-v7a` to be selected as the image which can be run without KVM. [This project](https://github.com/thyrlian/AndroidSDK#vnc) has a no-KVM emulator example which I managed to run successfully.

This would also require the `change_permission` function in `emulator.py` to not throw an exception if `/dev/kvm` is not available.

I understand that the emulator would be slow without KVM, but I believe it would make the project much more versatile without many changes.

### Probelm to solve

Running the emulator without KVM

**Comment by Tpuljak** [comment]
Thanks for the reply @budtmo.

The experience is not great 😅 
It is really slow, but, unfortunately, I have to run an emulator with Appium without KVM. Your project is great because it already includes Appium and NoVNC. I just wished it was flexible enough to allow the SYS_IMG as a parameter and not throw if kvm is unavailable.

From your perspective, you could just throw out a warning that the emulator would be really slow if KVM is not available.

**Comment by budtmo** [comment]
Hi @Tpuljak ,

if you want to run automation test using Appium using that system image, you might have flaky test unless you set a custom long timeout to wait the element. besides, the time execution will take hours just to run few test. I am not sure if it is what you want.

**Comment by Tpuljak** [comment]
@budtmo, yeah I understand. I might fork your repo and try to implement it myself and let you know how it goes. If I can manage to find a viable setup, I'll contribute with a PR.
